CHANGELOG.md

```diff
@@ -346,3 +341,5 @@ Released with 1.0.0-beta.37 code base.
 
 1. `create` function does not take in the optional parameter `entropy`
 2. `ignoreLength` will be removed as an optional parameter for `privateKeyToAccount`
+3. The `Wallet` no more supports address/number indexing. Have to use `wallet.get` instead.
+4. `Wallet.create` function doesn't accepts `entropy` param
```

Comment:
Would suggest to introduce such sensitive breaking change by creating an issue and linking it to PR. Referring back to a comment on PR is not an effective way to trace a breaking change.

Reply:
https://github.com/ChainSafe/web3.js/issues/4475#issuecomment-991093194